aptos-move/e2e-move-tests/src/tests/aggregator_v2.rs

```diff
@@ -0,0 +1,158 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    aggregator_v2::{
+        check, initialize, materialize, materialize_and_try_add, materialize_and_try_sub, new,
+        try_add, try_add_and_materialize, try_sub, try_sub_add, try_sub_and_materialize,
+    },
+    assert_abort, assert_success,
+    tests::common,
+    MoveHarness,
+};
+use aptos_language_e2e_tests::account::Account;
+use aptos_types::transaction::SignedTransaction;
+
+fn setup() -> (MoveHarness, Account) {
+    initialize(common::test_dir_path("aggregator.data/pack"))
+}
+
+#[test]
+fn test_aggregators_e2e() {
```

Comment:
This is the same as old aggregator tests, let's not duplicate test code. If you want to use the different files for V1 and V2, let's put the common code somewhere, e.g. you can have an array of function pointers you want to test, or use macros, etc.

Reply:
I agree the code looks unfortunately duplicate. But the function they call are actually different (aggregator.rs calls functions from aggregator module and aggregator_v2.rs calls functions from aggregator_v2 module). Using function pointers and macros for all of these seems to make the code less readable. We will deprecate the aggregator v1 code soon anyways right?

Reply:
Hmm, maybe. I do not like copying cause this way we can copy some mistakes done in V1 and then we would need to fix both😂. Let's add at least a TODO?